src/javis_viewer.jl

```diff
@@ -286,3 +286,107 @@ function _pluto_viewer(video::Video, frames::Int, objects::Vector)
     arr = collect(get_javis_frame(video, objects, frame) for frame in 1:frames)
     return arr
 end
+
+"""
+    StreamConfig
+
+Holds the conguration for livestream, defaults to `nothing`
+
+#Fields
+- `livestreamto::Symbol` Livestream platform `:local` or `:twitch`  
+- `protocol::String` The streaming protocol to be used. Defaults to UDP
+- `address::String` The IP address for the `:local` stream(ignored in case of `:twitch`)
+- `port::Int` The port for the `:local` stream(ignored in case of `:twitch`)
+- `twitch_key::String` TWITCH API key for your account
+"""
+struct StreamConfig
+    livestreamto::Symbol
+    protocol::String
+    address::String
+    port::Int
+    twitch_key::String
+end
+
+"""
+    setup_stream(livestreamto=:local; protocol="udp", address="0.0.0.0", port=8080, twitch_key="")
+
+Sets up the livestream configuration
+"""
+function setup_stream(livestreamto::Symbol = :local; protocol::String = "udp", address::String = "0.0.0.0", port::Int = 8080, twitch_key::String = "")
+    StreamConfig(livestreamto, protocol, address, port, twitch_key)
+end
+
+"""
+    cancel_stream()
+
+Kills the livestreaming process
```

Comment:
Add optional variable to kill online (i.e twitch processes) ffmpeg instances

Reply:
@Wikunia upon discussion with @TheCedarPrince I realise that there are many ways to improve this function.  Eg: We can use something like `lsof -i -P -n | grep ffmpeg`which scans all the listening ports on the local network rather than searching from all the active processes, but this won't work in case of twitch streaming, which can be solved by taking an optional stream config parameter into the function.

Also the current method of searching for ffmpeg streaming process creates problems with multiple streams(streaming to different animations/addresses/ports/protocols). Currently that isn't allowed as we cancel the ongoing stream before creating a new one to avoid race condition. If we want to support that in the future me might need to improve the searching methods for ffmpeg process.